README.md

```diff
@@ -8,6 +8,97 @@ software below.
 
 ----------------------------------------------------------------------
 
+Installation Instructions
+
+Linux (Ubuntu)
+1) Clone into the lomap repository
+    Navigate to desired location
+    Run the following line in shell:
+      git clone https://github.com/wasserfeder/lomap.git
+
+2) Install Spot using Debian Packages (https://spot.lrde.epita.fr/install.html)
+  Run the following lines in shell:
+    wget -q -O - https://www.lrde.epita.fr/repo/debian.gpg | apt-key add -
+    echo 'deb http://www.lrde.epita.fr/repo/debian/ stable/' >> /etc/apt/sources.list
+    apt-get update
+    apt-get install spot libspot-dev spot-doc python3-spot # Or a subset of those
+
+3) Install necessary dependencies:
+  Run the following lines in shell:
+    apt install python-pip
+    pip2 install matplotlib
+    pip2 install numpy
+    apt-get install python-tk
+    pip2 install networkx==1.11
+    pip2 install pyyamlCollecting pyyaml
+    pip2 install pp
+    apt install cmake
+
+    (Note: ensure that Python 2.7 is installed)
+
+  4) Download and unpack ltl2dstar
+    Download from: https://www.ltl2dstar.de/
+    Navigate to the ltl2dstar folder
+    Run the following lines in shell:
+      mkdir build
+      cd build/
+      cmake -DCMAKE_BUILD_TYPE=Release ../src
+      make
+      less ~/.bashrc
+      mkdir ~/bin
+      cp ltl2dstar ~/bin/
+      echo 'export PATH="$PATH:$HOME/bin"' >> ~/.bashrc
```

Comment:
Add comment to say that these last three steps are optional. They just ensure that ltl2dstar is in a path (in this case `~/bin`) that is searched for executables (i.e., `$PATH`).